METRICS.md

```diff
@@ -21,12 +21,18 @@ _All metrics start with `eradius_` prefix and the prefix is not included into ta
 | replies_total                                 | [$NAME, $IP, $PORT, $NASID, $NASIP] | counter   |
 | access_requests_total                         | [$NAME, $IP, $PORT, $NASID, $NASIP] | counter   |
 | accounting_requests_total                     | [$NAME, $IP, $PORT, $NASID, $NASIP] | counter   |
+| accounting_requests_start                     | [$NAME, $IP, $PORT, $NASID, $NASIP] | counter   |
```

Comment:
Prometheus likes counters to have suffix _total, like replies_total. Perhaps accounting_requests_start_total is a good name? Similar for the other new counters.

Reply:
I was take a look to the `client_accounting_responses` counter and then made a decision create without `_total` prefix, I not sure that without suffix `_total`  the Prometheus will work worse. But if `counter` without suffix `_tota` looks like confusing I will change it.

Reply:
@ebengt yes, you're right here, but let's make this in separate PR as well, because as @vkatsuba mentioned we also should rename a couple of metrics that are not related to this PR

Reply:
So first introduce new counter client_accounting_responses here. Then make a new pull request to change the name to client_accounting_responses_total?

Reply:
the real question is why is that a separate metric to begin with. Should the accounting type be a tag on accounting_requests_total?

Reply:
Add labels into `accounting_requests_total`. @ebengt, @RoadRunnr, @0xAX, please review.